CityOfDetroit/bloom
# 1321/listing section 8 details

## Issue
- Addresses #1321 

## Description
This PR addresses issue #1321 by adding messaging to a listings detail view when they have accept Section 8 Housing Choice Vouchers. This new messaging is included underneath the summary table along with the HMI section. ([Design](https://www.figma.com/file/D2fNPS5Ucll8uz7hyMUQGB/bloom-public-single-listing?node-id=2373%3A20712) ) Lastly, this PR removes the current household max income text under the rental summary table.
 
NOTE: 
1.  In order to unblock this PR's development, I merged in the section-8-partners-update branch so that I could utilize the new field.
2. This issue also referenced public tests but there is currently no infrastructure in place for that and it highlighted a potential gap in testing. This seems to require a separate discussion for the team to have.

## How Can This Be Tested/Reviewed?

This can be tested by first going to the Detroit partner's portal, clicking into a listing, and indicating that that listing accepts housing vouchers. Also, adding information to necessitate the HMI section. Then visit this listing on the public side and notice the two messages with 'Section 8 Housing Choice Voucher' bolded. For further testing, you can select no for section 8 on the partners side and verify that is no longer visible.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
